TestPlaybooks/playbook-Wildfire_Test.yml

```diff
@@ -1,93 +1,248 @@
 id: Wildfire Test
 version: -1
 name: Wildfire Test
+description: |-
+  A test playbook for Wild Fire.
+  Does not test the deprecated detonate commands.
 starttaskid: "0"
 tasks:
   "0":
     id: "0"
-    taskid: 983ae628-90f8-41b1-8ed7-85e29b591449
+    taskid: b36965cb-7fe9-4365-87e2-d240d91fd20e
     type: start
     task:
-      id: 983ae628-90f8-41b1-8ed7-85e29b591449
+      id: b36965cb-7fe9-4365-87e2-d240d91fd20e
       version: -1
       name: ""
       iscommand: false
       brand: ""
     nexttasks:
       '#none#':
-      - "4"
-      - "5"
+      - "10"
+    separatecontext: false
     view: |-
       {
         "position": {
-          "x": 254,
-          "y": -3
+          "x": 50,
+          "y": 50
         }
       }
-  "4":
-    id: "4"
-    taskid: d2c376b4-580e-4477-89f7-5ba40ff15fce
+    note: false
+  "6":
+    id: "6"
+    taskid: 57b392c9-4a9e-4c68-8adf-b137989e42f3
     type: regular
     task:
-      id: d2c376b4-580e-4477-89f7-5ba40ff15fce
+      id: 57b392c9-4a9e-4c68-8adf-b137989e42f3
       version: -1
-      name: WildFire Detonate File
-      description: Detonate file through WildFire
-      script: WildFire|||detonate-file
+      name: wildfire-upload-file-remote
+      description: Upload a file hosted on a website to WildFire for analysis
+      script: WildFire_2fix|||wildfire-upload-file-remote
```

Comment:
script name should be WildFire
did you forget to remove the `_2fix` part?
same for rest of the file